<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="findCornersButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2.1 Find Corners</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findIntrinsticButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2.2 Find Intrinsic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findExtrinsicButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2.3 FindExtrinsic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findDistrotionButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2.4 Find Distortion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showResultButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>2.5 Show result</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="indexSelector">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
